public/locales/en/graph.json

```diff
@@ -0,0 +1,17 @@
+{
+    "monday":"Mon",
+    "tuesday": "Tue",
+    "wednesday": "Wed",
+    "thursday": "Thu",
+    "friday": "Fri",
+    "saturday":"Sat",
+    "sunday": "Sun",
+    "noSensor": "Click a sensor to see its data!",
+    "time": "Time",
+    "good": "Good",
+    "moderate": "Moderate",
+    "sensitive": "Unhealthy For Sensitive Groups",
+    "unhealthy": "Unhealthy",
+    "very": "Very Unhealthy",
+    "hazardous": "Hazardous"
```

Comment:
I added these here even though they exist in another namespace because I didn't want the AQI ranges attached ex: "Good (0 - 50)"

Reply:
Are they also in the aqi table namespace without ranges?